### PR TITLE
Added the files and directory from Sphinx installation

### PR DESCRIPTION
**Pull request by jessehanson**
Installed Sphinx locally which added the build and source directories, as well as the makefile information. We will need to discuss certain preferences as a team and I will update the conf.py file accordingly.

### CODE REVIEW

**Comment by kwhitehall** [comment]
Thanks @jessehanson. Can you please push these to the docs folder?

**Comment by kwhitehall** [comment]
Hey @jessehanson, apparently the lib import extension sphinx.ext.imgmath throws an error with Sphinx v1.3.1. 
Please update PR accordingly.

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Aerolyzer'
+copyright = u'2017, Jesse Hanson, Reilly Collins, Sophia Liu'
```
Comment (kwhitehall) [inline docs/source/conf.py]: @jessehanson  plz update the copyright and author info to say 'Aerolyzer Development Team'

**Comment by jessehanson** [comment]
There are a few other instances of 'author' in conf.py that have yet to be populated, I can set those to 'Aerolyzer Development Team' if you'd like.

**Comment by kwhitehall** [comment]
Thanks @jessehanson. LGTM!

**Pull request merged**